Source/DafnyLanguageServer/Language/VerificationResult.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Microsoft.Dafny.LanguageServer.Language {
+  /// <summary>
+  /// Data class to expose the program verification results.
+  /// </summary>
+  /// <param name="Verified"><c>true</c> if the program was successfuly verified. <c>false</c> if no verification was run or the program has errors.</param>
+  /// <param name="SerializedCounterExamples">The counter examples to disprove the program's correctness, serialized to a string.</param>
+  public record VerificationResult(bool Verified, string? SerializedCounterExamples = null);
```

Comment:
To me `Verified` sounds a lot like it passed verification, but I think here it only means that verification was ran. Could you consider renaming this to `VerificationRan` ?

Reply:
`Verified = true` is meant to denote that a program passed its verification. Do you prefer naming it `SuccessfullyVerified` to avoid misinterpretations? I've tried to use the same naming scheme as used in [VerificationCompletedParams](https://github.com/dafny-lang/dafny/blob/46ec9ce6102b84daad3d48c28c6d76e8beb0eebc/Source/DafnyLanguageServer/Workspace/Notifications/VerificationCompletedParams.cs#L12).

Reply:
Would it be possible to subdivide the "Not verified" into parse errors vs resolution/type-checking errors? If so, would the following 5 phrases be short enough to display in the status bar?
* `Parse errors`
* `Type errors`
* `Verifying...`
* `Verification errors`
* `Verification success`

Reply:
That's a great idea. We can display messages of arbitrary length. However, I try to avoid long texts as I am certain that some people maximize the use of the status bar in a way where every inch counts.

Reply:
We could distinguish between `Resolution errors` and `Type errors`, but I think most users wouldn't really understand the difference. That's why I just suggested `Type errors`.

When verification is disabled, could we display `Verification off` or `Verification disabled`?

I appreciate that there's not much room in the status bar. Since `Verification is such a long word`, it would be nice to reverse the order of the words, like `Success verification` instead of `Verification success`. However, I fear that would not be as readable. Maybe there are other words that can be used.